plugins/global-search/src/utils/useFocus.ts

```diff
@@ -0,0 +1,36 @@
+import type { FocusableElement } from "@react-types/shared"
+import { useCallback } from "react"
+import { useFocusManager } from "react-aria"
+
+export function useFocusHandlers() {
+    const focusManager = useFocusManager()
+
+    const onKeyDown = useCallback(
+        (event: React.KeyboardEvent<HTMLElement>) => {
+            const isResultMatch = (element: Element) =>
+                element.id === "text-search-input" || element.matches("[data-result-match]")
```

Comment:
There must be a non-stringly-typed way to do this, no? 🤔 Via some sort of ref collection maybe?

Reply:
I was thinking on that, but didn't really felt it's worth it to add that complexity.

Reply:
Changed it to something less loose – curious what you think